yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -801,10 +801,11 @@ func generateAssessmentReport() (err error) {
 
 func getAssessmentReportContentFromAnalyzeSchema() error {
 	schemaAnalysisReport := analyzeSchemaInternal(&source)
+	assessmentReport.MigrationComplexity = schemaAnalysisReport.MigrationComplexity
 	assessmentReport.SchemaSummary = schemaAnalysisReport.SchemaSummary
-	assessmentReport.SchemaSummaryDBObjectsDesc = "Objects that will be created on the target YugabyteDB."
+	assessmentReport.SchemaSummary.Description = "Objects that will be created on the target YugabyteDB."
```

Comment:
This can now also go in analyze-schema now right?

Reply:
yeah but i thought to follow - keep the description of a struct inside that struct itself.

Reply:
@sanyamsinghal, No, I meant we are not populating the `SchemaSummary.Description` in the analyze schema report. So should we that as well ?